specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/table.json

```diff
@@ -207,6 +210,9 @@
         "x-ms-examples": {
           "TableOperationPatch": {
             "$ref": "./examples/TableOperationPatch.json"
+          },
+          "TableOperationPutOrPatchAcls": {
+            "$ref": "./examples/TableOperationPutOrPatchAcls.json"
```

Comment:
This API input parameter only has "#/parameters/TableName", so the swagger not support patch table with signedIdentifiers.

Reply:
Added